### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'read'

### ISSUE THREAD

**Issue by grihabor** [eywalker/nvidia-docker-compose#18]
Installed nvidia-docker-compose via  
```
sudo pip3 install nvidia-docker-compose
```  
Executed ```nvidia-docker-compose``` and got an error:  
```
Traceback (most recent call last):
  File "/usr/local/bin/nvidia-docker-compose", line 64, in <module>
    config = yaml.load(args.file)
  File "/home/grihabor/.local/lib/python3.5/site-packages/yaml/__init__.py", line 70, in load
    loader = Loader(stream)
  File "/home/grihabor/.local/lib/python3.5/site-packages/yaml/loader.py", line 34, in __init__
    Reader.__init__(self, stream)
  File "/home/grihabor/.local/lib/python3.5/site-packages/yaml/reader.py", line 85, in __init__
    self.determine_encoding()
  File "/home/grihabor/.local/lib/python3.5/site-packages/yaml/reader.py", line 124, in determine_encoding
    self.update_raw()
  File "/home/grihabor/.local/lib/python3.5/site-packages/yaml/reader.py", line 178, in update_raw
    data = self.stream.read(size)
AttributeError: 'NoneType' object has no attribute 'read'
```

**Comment by grihabor** [comment]
Solved the problem: this error happened because there was no docker-compose.yml file

**Comment by eywalker** [comment]
Glad you figured out! But this certainly indicates that the tool could use better error message for such a common case.

**Comment by eywalker** [comment]
Closing this issue as resolved but will improve error messaging via #19
